processes/releasing.md

```diff
@@ -90,32 +93,37 @@ components that are tested in periodic jobs.
 We announce the release in Kubernetes slack on `#cluster-api-baremetal` channel
 and through the `metal3-dev` group mailing list.
 
-Email template:
+Email template, with title `New CAPM3, IPAM and BMO minor releases are out!`
+(adjust as necessary depending on what is being released):
 
 ```text
 Hey folks,
 
-CAPM3 v1.x.y, IPAM v1.x.y and BMO v0.x.y minor releases are out now!
-Release notes can be found here: [1], [2], [3].
+CAPM3 v1.x.y, IPAM v1.x.y, BMO v0.x.y and Ironic-image v24.x.y minor releases are out now!
+Release notes can be found here: [1], [2], [3], [4].
 
 Thanks to all our contributors!
 
 [1] https://github.com/metal3-io/cluster-api-provider-metal3/releases/tag/v1.x.y
 [2] https://github.com/metal3-io/ip-address-manager/releases/tag/v1.x.y
 [3] https://github.com/metal3-io/baremetal-operator/releases/tag/v0.x.y
+[4] https://github.com/metal3-io/ironic-image/releases/tag/v24.x.y
+
+Thanks to all our contributors!
 ```
 
 Slack template:
 
 ```text
 Hey folks,
 
-CAPM3 v1.x.y, IPAM v1.x.y and BMO v0.x.y minor releases are out now :tada::metal3:!
+New Metal3 project releases are out now :tada::metal3:!
```

Comment:
I would rather keep it as it was, we are not doing project release as per say.

Reply:
Sure, I changed it back.